articles/node.js-vs-python/index.md

```diff
@@ -0,0 +1,123 @@
+
+Node.js is cross-stage. This means that a developer can create one desktop app that will work On different operating systems like MAC OS, Linux, and Windows.  This will be a great advantage to the developer in terms of the project cost.
+
+#### Python
+Python is a full-stack language. It is used in backend development while its frameworks are used in frontend development. 
+
+A python program can be written in MAC OS and the same program run in Linux, therefore Python is also a cross-stage
+
+Python is a good language for web development as well as desktop development, but unlike Node.js it is never used in mobile app development. 
+
+### Extensibility
+ Extensibility is the ability to add more features and functionalities to a technology. The more frameworks you add to a backend language, the more it increases its functionality.
+### Node.Js
+The use of Node.js and JavaScript together has led to an increase in its popularity. More frameworks have been created with Node.js being the backbone. These Frameworks include:
+1. [Loopback.js](https://loopback.io/doc/)- to make dynamic end-end REST APIs with no coding.
+2. [DerbyJS](https://derbyjs.com/) - to make web applications
+3. [Hapi.js](https://simpleprogrammer.com/introduction-hapijs/)- for creating web services such as REST APIs
+
+### Python
+After the introduction of Python, a lot of frameworks and development tools have been created.
+
+For instance, Python can be incorporated with code editors like Sublime Text, which offers some additional editing features and punctuation extensions.
+
+The great extensibility of Python and the use of many frameworks have made Python be such a great backend language that every developer would desire to use. Python frameworks include:
+
+1. [Django](https://docs.djangoproject.com/en/3.2/)
+
+2. [Flask](https://flask.palletsprojects.com/en/1.1.x/)
+
+3. [Web2Py](https://loopback.io/doc/)
+
+### Architecture
+Another important feature that a developer should give a priority before choosing the backend language is the architecture. They are two types of architecture:
+
+- Synchronous - The job is first completed before accepting another request.
+- Asynchronous - Here, more than one process can run at the same time.
+
+### Node.Js
+Node.js enables asynchronous. A process can be called even if another event is occurring. This means that no process is ever blocked. The different processes can be processed.
+
+Node.js is event-driven(can respond to the user in various ways). Therefore it is suitable for the development of games.
+
+### Python
+Python is not event-driven. To build an event-driven app using Python, you need to install a tool like [CPython](https://stackoverflow.com/questions/17130975/python-vs-cpython).
+
+Python enables synchronous. This makes it slow and therefore it is not suitable for projects that require a high speed of execution. So if you want to create a game program you'd rather not choose Python but Node.js
+
```

Comment:
Correct me if I'm wrong but, although less common, can't Python also carry out an asynchronous paradigm? Rephrase this statement to reflect such.